packages/calcite-components/src/demos/input-number.html

```diff
@@ -135,8 +135,14 @@
         <div class="child-flex font right-aligned-text">Number Vertical</div>
         <div class="child-flex">
           <calcite-label scale="s">
-            Input Label
-            <calcite-input-number placeholder="Placeholder" scale="s" value="123" step="1"></calcite-input-number>
+            Input Label AR2
```

Comment:
@josercarcamo for consistency, could you restore the previous HTML and either:

* move this particular Arabic configuration to a new section or
* omit it for now?